curp/src/server/raw_curp/log.rs

```diff
@@ -43,14 +43,135 @@ pub(super) struct Log<C: Command> {
     pub(super) last_exe: LogIndex,
     /// Tx to send log entries to persist task
     log_tx: mpsc::UnboundedSender<LogEntry<C>>,
-    /// Prefix sum vector of log entries
-    batch_index: VecDeque<u64>,
     /// Batch size limit
     batch_limit: u64,
```

Comment:
`batch_limit` just used in the `LogEntryVecDeque`, I recommend move it to `LogEntryVecDeque`.

Reply:
lgtm